web/src/main/java/org/cbioportal/web/mixin/summary/GeneticDataSummaryMixin.java

```diff
@@ -1,13 +1,18 @@
 package org.cbioportal.web.mixin.summary;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 public class GeneticDataSummaryMixin {
 
     @JsonIgnore
     private Integer geneticProfileId;
+    @JsonProperty("geneticProfileId")
```

Comment:
is this the internal id? I will test, but if we are exposing internal ids to external clients, we should discuss soon, and at least we should give prominent visual cautions on the API that the returned internal ids are not stable (they may change on every database update)